skel/bounty-run

```diff
@@ -33,7 +33,7 @@ BWRAPBOX_RLIMIT_AS_HIGH=$((768*1024*1024))
 BWRAPBOX_RLIMIT_AS_MAX=$((1024*1024*1024))
 
 # cpu time (in seconds)
-BWRAPBOX_RLIMIT_CPU_MAX=150
```

Comment:
Because we shouldn't need it so high if we're not compiling `forge-std`.

Reply:
Hmmm... The Adder project is so simple.
I think we have to test with another project, a more realistic one, and reevaluate it. 😉

Reply:
Fair enough. I guess we can try to compile a more complex/sizeable project, like `rollups-contracts`.

Reply:
Compiling the `rollups-contracts` project took 48 seconds.
How about raising it to 60 seconds (aka 1 minute)?

Reply:
Nice test!!!
Considering that we will let anyone submit their project to run and it can be anything (even uniswap contracts, for example), I would let 150s to make it harder to someone receive a timeout. 😉

Reply:
What do you think, @edubart?

Reply:
The first usage of it will be on testnet. We can monitor the usage to set the best config before promoting to mainnet.
I will use your branch to review the tutorial.

Reply:
Reverted.